contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -528,7 +528,7 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
         emit CancelUnstake(msg.sender);
     }
 
-    function unstake() external {
+    function unstake() external notPaused {
         require(unstaking[msg.sender] == true, "Account has not triggered unstake cooldown");
         require(
             lastUnstakeTime[msg.sender] < block.timestamp.sub(unstakeDurationPeriod),
```

Comment:
updateVolume in line 548 intenitionally left without pause modifier.
To remain the AMM volume accumulation.
Add pause modifier if you think otherwise